epochStart/shardchain/trigger_test.go

```diff
@@ -726,3 +727,124 @@ func TestTrigger_AddMissingValidatorsInfo(t *testing.T) {
 	assert.Equal(t, uint32(1), epochStartTrigger.mapMissingValidatorsInfo["c"])
 	epochStartTrigger.mutMissingValidatorsInfo.RUnlock()
 }
+
+func TestTrigger_SaveEpochStartInfoToStaticStorage(t *testing.T) {
+	t.Parallel()
+
+	epoch := uint32(3)
+
+	args := createMockShardEpochStartTriggerArguments()
+	args.Epoch = epoch
+
+	mbHeaderHash1 := []byte("mbHeaderHash1")
+	epochStartHeader := &block.MetaBlock{
+		Nonce: 100,
+		Round: 100,
+		Epoch: epoch,
+		MiniBlockHeaders: []block.MiniBlockHeader{
+			{
+				Hash: mbHeaderHash1,
+				Type: block.PeerBlock,
+			},
+			{
+				Hash: []byte("mbHeaderHash3"),
+				Type: block.TxBlock,
+			},
+		},
+	}
+	headerBytes, _ := args.Marshalizer.Marshal(epochStartHeader)
+
+	txHash1 := []byte("txHash1")
+	txHash2 := []byte("txHash2")
+	mb := &block.MiniBlock{
+		TxHashes: [][]byte{txHash1, txHash2},
+		Type:     block.PeerBlock,
+	}
+
+	validatorInfo1 := &state.ShardValidatorInfo{
+		PublicKey:  []byte("pubKey1"),
+		ShardId:    0,
+		List:       "eligible",
+		Index:      1,
+		TempRating: 11,
+	}
+	validatorInfo2 := &state.ShardValidatorInfo{
+		PublicKey:  []byte("pubKey2"),
+		ShardId:    0,
+		List:       "eligible",
+		Index:      2,
+		TempRating: 12,
+	}
+
+	putCalls := 0
+	args.Storage = &storageStubs.ChainStorerStub{
+		GetStorerCalled: func(unitType dataRetriever.UnitType) (storage.Storer, error) {
+			if unitType == dataRetriever.MetaBlockUnit {
+				return &storageStubs.StorerStub{
+					GetCalled: func(key []byte) (bytes []byte, err error) {
+						return headerBytes, nil
+					},
+				}, nil
+			} else if unitType == dataRetriever.EpochStartStaticUnit {
```

Comment:
I think we can simply have an if here, as it returns on the above condition

Reply:
right, changed